api/app/settings/common.py

```diff
@@ -390,6 +390,7 @@
 ]
 
 AUTHENTICATION_BACKENDS = [
```

Comment:
This backend will never be used if ldap is installed. see: https://github.com/Flagsmith/flagsmith/blob/main/api/app/settings/common.py#L1158

Reply:
The backend is now removed.